api_v3/services/UserService.php

```diff
@@ -417,7 +417,9 @@ public function loginDataResetPasswordAction($loginDataId, $newPassword)
 			}
 			throw $e;
 		}
-		return $updateLoginData;
+
+		$user = KuserPeer::getKuserByEmail($loginDataId, kCurrentContext::$partner_id);
```

Comment:
i think it will make more sense to extract the user before we update the login data, and also throw exception like we do in enable login in case it is not found

Reply:
Done